src/Core/Grid/Query/Filter/SqlFilters.php

```diff
@@ -35,6 +37,7 @@ final class SqlFilters
     public const WHERE_LIKE = 2;
     public const HAVING_LIKE = 3;
     public const WHERE_DATE = 4;
+    public const MIN_MAX = 5;
```

Comment:
wouldn't `range` be better instead of `min max`?

Reply:
I guess range would make a bit more sense. But min max isn't exactly confusing and we have `IntegerMinMaxFilterType` and `NumberMinMaxFilterType`. In grid it appears as two inputs min and max, not as range.

So for the sake of consistency I would like to keep MIN_MAX.